frontend/app/models/team.js

```diff
@@ -7,10 +7,24 @@ export default class Team extends Model {
   @attr("boolean") private;
   @attr("boolean") favourised;
   @attr("boolean") deletable;
+  @attr("number") unread_count;
   @hasMany("folder") folders;
   @hasMany("teammember") teammembers;
 
   get isPersonalTeam() {
     return this.type === "Team::Personal";
   }
+
+  get unreadTransferredFilesInFolders() {
```

Comment:
also here maybe use a method name like "unreadTransfersInFolder" to generalize it for credentials share, then we don't have to rename anything when implementing the credential share feature :wink: 

Also is this supposed to be plural, because you only check inbox folder right?